oobi/benchmark-definitions/python/data_types.py

```diff
@@ -48,12 +73,141 @@ class Model(object):
   name: str
   # Tags that describe the model characteristics.
   tags: List[str]
-  framework_type: FrameworkType
+  framework_type: ModelFrameworkType
   # Source of the model implementation.
   source_info: str
   input_batch_size: int
   inputs: ModelData
   outputs: ModelData
+  # A list of artifacts derived from this model.
+  artifacts: List[ModelArtifact]
 
   def __str__(self):
     return self.name
+
```

Comment:
It looks like the classes below are not used right now. I'm still prototyping them. Maybe we can remove them now and I'll add back later.

Reply:
Removed